contrib/gis/src/main/java/org/apache/drill/exec/expr/fn/impl/gis/STAsJSON.java

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
@ChrisSandison Apache header should be in a from of comment. Please here and in other PR.